test_queue.py

```diff
@@ -17,12 +17,14 @@ def make_popd_q():
     q.enqueue('asdf')
     return q
 
+
 def test_node_init():
     b = Node(u'a')
     assert b.data == u'a'
     c = Node(u'a', b)
     assert c.next.data == u'a'
 
+
 def test_queue_init(make_empty_q):
```

Comment:
Be sure to use docstrings, and when you do surrond the in triple-quotes. This makes them accessible by the __doc__ function.